listenbrainz/webserver/static/css/recommendation-page.less

```diff
@@ -0,0 +1,87 @@
+
+
+#recommendations {
+    padding: 10px;
+
+    > .recommendation-card {
+      padding: 15px;
+      margin-bottom: 7px;
+      max-height: 72px;
+
+      &.current-recommendation {
+        background-color: rgba(217, 237, 247, 0.3) !important ;
+      }
+
+       div {
+         padding: 0px;
+       }
+
+      > .rec-col-xs-9, .rec-col-xs-12 {
+
+        > .rec-col-xs-9, .rec-col-xs-12 {
+          display: flex;
+
+          .rec-details {
+            width: 100%;
+
+            p {
+              overflow: hidden;
+              text-overflow: ellipsis;
+              white-space:nowrap;
+              margin-bottom: 0px;
+            }
+          }
+        }
+
+      }
+
+
+      > .rec-col-xs-3 {
+        display: table;
+        height: 40px;
+
+        > * {
+          display: table-cell;
+          vertical-align: middle;
+        }
+
+        a {
+          word-break: break-word;
+        }
+
+        .recommendation-controls {
```

Comment:
![image](https://user-images.githubusercontent.com/26141595/95719720-dc3f7380-0c8d-11eb-808f-c4cdac4b22f0.png)

Is this because of recommendation-control class?